sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md

```diff
@@ -20,7 +20,8 @@ irregular-plural-words:
   quota: quota
 
 rename-mapping:
-  LoadTestResource: LoadTesting
+  Resource: LoadTestingBaseResource
+  LoadTestResource: LoadTestingResource
   QuotaResource: LoadTestingQuota
   CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailabilityResponse
   EncryptionProperties: CustomerManagedKeyEncryptionProperties
```

Comment:
OutboundEnvironmentEndpointCollection: OutboundEnvironmentEndpointPageList ?

Reply:
Here the `Collection` is a reserved suffix for ResourceCollection, so here rename it, and the naming pattern is change the suffix from `Collection` to `ListResult`.